index.md

```diff
@@ -10,18 +10,60 @@ layout: standard-page
 Good humored, cooperative, competitive, consensual torture, for the purposes of
 exploring strategies, design, progress and history of gaming
 
-### Members
-* There is no single leader of the Zargon community. Zargon is a community run
-  effort. All members are encouraged to participate in this effort.
-* You may invite new members to join in gaming, but they are your
-  responsibility.
-* New invitees should come to provisional games (e.g., Friday games, or similar
-  virtual games) for the sake of the invitee and the community.
-* If community members at large do not like a guest then that guest may not be
-  invited to future events.
+## Membership
+There is no single leader of the Zargon community. Zargon is a community run
+effort. All members are encouraged to participate in this effort.
+
+The following guidelines are intended to allow us to maintain an inclusive
+environment where all members feel comfortable gaming with any other member of
+the group rather than fracturing in cliques. They are enumerated in detail so
+as to mitigate subjectivity in role assignment.
+
+### Obligations
 * Members are expected to maintain basic personal hygiene.
 * Members are expected to be gracious and respect additional restrictions of
-  venue (e.g. no smoking inside; no young children; allergy considerations).
+  venue
+    - e.g. no smoking inside; no young children; allergy considerations.
+
+### Active Members
+* An active member is one who regularly does one or more of the following
+  - Attends and particpates in scheduled gaming
+  - Attends and participates in social events
+  - Participates in virtual group conversations (e.g. text, voice, video)
+* Active members are noted by the **Zargonaut** role in the group discord.
+
+### Contributing Members
+* A contributing member is one who regularly does one or more of the following
+  - Schedules game sessions
+  - Schedules social events
+  - Brings new games for the group to try
+  - Teaches games at gaming events
+  - Creates content to aid the group (e.g. rules handouts, character
+    illustrations, game storage solutions)
+* An active member who feels they are making sufficient contributions should
+  bring this to the attention of one of the contributing members.
+* Contributing members are noted by the **Adept** role in the group discord.
+
+### New Members
+* Any active member in good standing may invite non-member to join social
+  or gaming events, but take responsibility for their behavior.
```

Comment:
```suggestion
* Any active member in good standing may invite a non-member to join social
  or gaming events, but please take responsibility for their behavior.
```

Reply:
Personally I would change `please` to `must` there.

Reply:
+1 to must.  Maybe "should" if we want it to sound more polite.  But the word "please" moves the sentence from the third person to the second person imperative.

Reply:
i chose "should" because it's more polite sounding.

Reply:
Keeping both off to keep the tone better.